### ISSUE TITLE
Save games copied to iOS aren't seen

### ISSUE THREAD

**Issue by mutantant** [alexbatalov/fallout1-ce#4]
When copying the data folder with an active save game inside ("data/savegame/slot01"), seen in the desktop version, the iOS shows a blank load screen without the save populating the list.

**Comment by alexbatalov** [comment]
Check `fallout.cfg` on iOS, make sure `master_patches` match case (I have `master_patches=data`). Everything inside `data` dir should be uppercased on every platform, but generally you shouldn't touch anything inside. Once everything is setup, you move saves between iOS and Mac by copying `data` dir (not very convenient, I'm investigating other options)

**Comment by mutantant** [comment]
Solved this in a way: master_patches was correct, BUT the engine sees DATA and data as different, though it will read game information from both (as in I had it as DATA, and there were no errors on loading), BUT will not see the savegames in the DATA folder. In fact, it made its own "data" folder next to my "DATA" folder where it constructed a new file structure and saved the new save I made in slot 2 in iOS.

So either it loaded DATA files no problem, or it simply didn't see them and the game only loaded the info and files from the dat files, ignoring the DATA directory without error, I'm not sure which.

Needless to say, once I changed DATA to data, my old save was showing. I'm assuming I may need to start a new game, however, as it's likely it sisn't load the scripts in the DATA filder at all and are running off older, unpatched scripts in that save.

In short, make sure the capitilization in the INI matches the file structure, I suppose :) Still no music, however, no matter what I try (different, though related, issue I've opened and you have replied to)
